Failed to minify the code from this file ./node_modules/getstream/src/lib/client.js:159

I'm having trouble compiling with `npm run-script build` when using the `getstream` library.

Steps to reproduce:

1) Download sample app from https://codesandbox.io/s/jvm683w6pw
2) `npm install`
3) `npm run-script build`

And the output:
```
 $ npm run-script build                                                                                                                                                 
> new@1.0.0 build /Users/jimmy/Downloads/jvm683w6pw
> react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/getstream/src/lib/client.js:159 

Read more here: http://bit.ly/2tRViJ9

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! new@1.0.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the new@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jimmy/.npm/_logs/2018-03-14T11_38_21_345Z-debug.log
```

As you can see the app is just a vanilla sample React app from https://codesandbox.io and the only addition was adding `getstream` as a dependency and `import stream from "getstream";`.

When trying with `UglifyJS` on my other project I seem to get a more verbose error message:
```
Unexpected character '`' [./~/getstream/src/lib/client.js:159,0][static/js/main.f24a4d28.js:94724,41]
```

--- a/src/lib/client.js
+++ b/src/lib/client.js
@@ -156,9 +156,9 @@ StreamClient.prototype = {
     var callback = this.wrapCallback(cb);
     return function task(error, response, body) {
       if (error) {
-        reject(new errors.StreamApiError(`${error}`, body, response));
+        reject(new errors.StreamApiError(error, body, response));
       } else if (!/^2/.test('' + response.statusCode)) {
-        reject(new errors.StreamApiError(`${JSON.stringify(body)} with HTTP status code ${response.statusCode}`, body, response));
+        reject(new errors.StreamApiError(JSON.stringify(body) + ' with HTTP status code ' + response.statusCode, body, response));
       } else {
         fulfill(body);
       }

